lib/Runtime/InternalPropertyList.h

```diff
@@ -6,7 +6,7 @@
 // They become nameless compile time known PropertyRecords, stored as static
 // fields on the InternalPropertyRecords class.
 
-INTERNALPROPERTY(TypeOfPrototypObject)  // Used to store the type of the prototype object in the prototype objects slots
+INTERNALPROPERTY(TypeOfPrototypeObject)  // Used to store the type of the prototype object in the prototype objects slots
```

Comment:
Since we're fixing a typo here I'm going to nitpick and request that we also keep the comments aligned